### DISCUSSION TITLE
24/7 recording and don't really care about events

### DISCUSSION THREAD

**Discussion by mforgues** [blakeblackshear/frigate#7484] [Ask A Question] [answered]
I wanted to use frigate simply as a recorder for two cameras I have. I would like to record 24/7 and keep maybe 7 days of recordings.

I understand that events are typically used to trigger recordings but can I disable the events and simply use frigate to record?

**Comment by NickM-27** [comment, marked as answer]
yes of course, just enable 24/7 recording in the config

**Reply by mforgues** [reply]
Something like this would work?
![Screenshot_20230815-185630](https://github.com/blakeblackshear/frigate/assets/30607071/9321c24d-3756-4f86-ac3f-a68d81135334)

**Reply by NickM-27** [reply]
I wouldn't recommend changing expire_interval
